schema/compose-spec.json

```diff
@@ -599,12 +602,12 @@
       "items": {
         "type": "object",
         "properties": {
-          "capabilities": {"$ref": "#/definitions/list_of_strings"},
-          "count": {"type": ["string", "integer"]},
-          "device_ids": {"$ref": "#/definitions/list_of_strings"},
-          "driver":{"type": "string"},
-          "options":{"$ref": "#/definitions/list_or_dict"}
-        },
+            "capabilities": {"$ref": "#/definitions/list_of_strings"},
+            "count": {"type": ["string", "integer"]},
+            "device_ids": {"$ref": "#/definitions/list_of_strings"},
+            "driver":{"type": "string"},
+            "options":{"$ref": "#/definitions/list_or_dict"}
+          },
```

Comment:
Indentation seems to be 2 characters in the rest of the file, was this change intended?

Reply:
I copied the exact content from compose-spec - we haven't yet found a way to directly refer to the reference repository for this file

Reply:
Maybe a JSON linter and the same approach as we use on `V1`, which is to integrate the download of the spec file in the build.